Airports/EGXC/Ground Map/Geo.txt

```diff
@@ -1,5 +1,4 @@
-; Start Coningsby (EGXC) Geo
-N053.05.19.860 W000.11.08.203 N053.05.21.617 W000.11.09.121 building
+EGXC Coningsby N053.05.19.860 W000.11.08.203 N053.05.21.617 W000.11.09.121 building
```

Comment:
```suggestion
EGXC Coningsby S999.00.00.000 E999.00.00.000 S999.00.00.000 E999.00.00.000
N053.05.19.860 W000.11.08.203 N053.05.21.617 W000.11.09.121 building
```

Reply:
It will work as originally committed -- having the S999.00.00.000 etc. just allows for backwards compatibility as it doesn't understand names within [GEO]. See #2493 for a proposal regarding this. 

If you want it changed then I'm happy but imo it adds clutter for no reason, as we build our sector file specifically for Euroscope.

Reply:
I just thought it best for it to match the other ones for now?